api/nodes/views.py

```diff
@@ -194,38 +194,38 @@ def get_queryset(self):
         return children
 
 
-class NodePointersList(generics.ListCreateAPIView, NodeMixin):
-    """Pointers to other nodes.
+class NodeLinksList(generics.ListCreateAPIView, NodeMixin):
+    """Node_links to other nodes.
```

Comment:
I think in the docstrings we should just call them `Node Links` unless it's referring specifically to a parameter or path component.